Assets/Plugins/Editor/Uplift/Common/UnityPackageOpener.cs

```diff
@@ -0,0 +1,133 @@
+using System;
+using System.IO;
+using System.IO.Compression;
+using System.Text.RegularExpressions;
+using SharpCompress.Archives;
+using SharpCompress.Archives.Tar;
+using UnityEngine;
+using Uplift.Extensions;
+
+namespace Uplift.Common
+{
+    public class UnityPackageOpener
+    {
+        public void OpenUnityPackage(string archivePath, bool deleteAfterOpening = false)
+        {
+            OpenUnityPackage(
+                archivePath,
+                Path.GetDirectoryName(archivePath),
+                deleteAfterOpening
+            );
+        }
+        public void OpenUnityPackage(string archivePath, string destinationPath, bool deleteAfterOpening = false)
```

Comment:
Is `deleteAfterOpening` needed? What's the use case?

Reply:
None actually. Removing it